templates/login.html

```diff
@@ -14,11 +14,54 @@
 }
 </style>
 {% endblock %}
+{% block scripts%}
+<script type="text/javascript">
+
+function searchform() {
+  var bookload = "<br><br><h3 style = 'text-align:center'>Books are loading......</h3>"
+  document.getElementById("searchresults").innerHTML=bookload;
+  var url = "/api/search";
+  var form = new FormData(document.getElementById("searchform"));
+  var text = form.get("search")
+  responseobj = {}
+  responseobj['query'] = text;
+  var json = JSON.stringify(responseobj);
+  var xhr = new XMLHttpRequest();
+  xhr.open("POST", url);
+  xhr.setRequestHeader('Content-Type','application/json');
+  xhr.send(json);
+  xhr.onload = function () {
+  var data = JSON.parse(xhr.responseText);
+  if (xhr.status == "200") {
+    var htmltext = "<br><div class = 'container'> <div class = 'tableFixHead'>"+
+    "<table class='table' border = 1 align = 'center' style = 'background-color: pink; color: black;'>"+
+    "<thead class='thead-dark' style='color: white;'><tr><th> {{ 'Isbn' }}</th><th> {{ 'Title' }}</th><th> {{ 'Author' }}</th>"+
+    "<th> {{ 'Year' }}</th><th> {{ ' Link to Book' }}</th></tr></thead><tbody>";
+    for(var i = 0; i < data.length; i++) {
+      htmltext = htmltext + "<tr>" + "<td>"+data[i]["isbn"] +"</td>"+ "<td>" + data[i]["title"] +"</td>"+
+      "<td>" + data[i]["author"] +"</td>"+ "<td>" + data[i]["year"] +"</td><td><button onclick = 'bookdisplay(this)' id = "+
+      data[i]["isbn"] + ">View Book</button></td></tr>"
+    }
+    htmltext = htmltext + " </tbody> </table> </div></div>"  
+    document.getElementById("searchresults").innerHTML=htmltext;
+  }
+  else {
+    if (xhr.status == 404) {
+      var errormessage = "<br><br><h3 style = 'text-align:center'>No Book Results found</h3>"
```

Comment:
Why aren't you using the error message from the API directly?

Reply:
Completed bro